internal/website/content/howto/pubsub/_index.md

```diff
@@ -4,14 +4,26 @@ date: 2019-03-26T09:44:06-07:00
 showInSidenav: true
 ---
 
+The [`pubsub` package][] provides an easy and portable way to interact with
+publish/subscribe systems. This guide shows how to work with pubsub
+in the Go CDK.
+
+<!--more-->
+
 The [publish/subscribe model][] allows parts of a system to publish messages
 that other parts of a system may subscribe to. This is commonly used to
 arrange for work to happen at some point after an interactive frontend
 request is finished or in other event-driven computing.
 
-The Go CDK provides portable APIs for publishing messages and subscribing to
+The [`pubsub` package][] supports operations to publish messages and subscribe to
```

Comment:
Nit: can you update this to say

...supports operations to publish messages to a topic, and to subscre to a topic to receive messages.

or something like that? (you don't "subscribe to a message", you subscribe to a topic).

Reply:
I reworded this here and in other places, including the title of the subscribe/ page.
(Searching on Google does show usage of "subscribe to messages" phrase though.)